frontend/src/components/TE/VersionDownloadLink.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <a v-if="downloadLink" :href="downloadLink" v-bind="$attrs"><slot></slot></a>
```

Comment:
What is `v-bind="$attrs"` for?

Reply:
That allows any attribute that's not a prop to be passed to the `a` tag through the component.